### ISSUE TITLE
Improve accessibilitytest.org result, 

### ISSUE THREAD

**Issue by 1peter10** [1peter10/minimola#5]
See https://accessibilitytest.org/results/5Drq0TXIRiPo for current result. The easy issues (contrast) should be solved.
